conf/scalardb-custom-values.yaml

```diff
@@ -14,42 +14,45 @@ envoy:
     type: LoadBalancer
     annotations:
       service.beta.kubernetes.io/azure-load-balancer-internal: "true"
-      service.beta.kubernetes.io/azure-load-balancer-internal-subnet: "k8s_ingress"
       service.beta.kubernetes.io/aws-load-balancer-internal: "true"
       service.beta.kubernetes.io/aws-load-balancer-type: "nlb"
 
-  resources: {}
+  resources:
+    requests:
+      cpu: 200m
+      memory: 256Mi
+    limits:
+      cpu: 300m
+      memory: 328Mi
 
   tolerations:
     - effect: NoSchedule
-      key: kubernetes.io/app
+      key: scalar-labs.com/dedicated-node
       operator: Equal
-      value: scalardbpool
+      value: scalardb
 
   affinity:
     nodeAffinity:
       requiredDuringSchedulingIgnoredDuringExecution:
         nodeSelectorTerms:
           - matchExpressions:
-              - key: agentpool
+              - key: scalar-labs.com/dedicated-node
```

Comment:
To make the purpose of this label clear, I renamed the key name of label.
This label is used for making specific worker nodes dedicated to Scalar products.